vyper/parser/expr.py

```diff
@@ -183,9 +186,9 @@ def number(self):
             )
         else:
             raise InvalidLiteralException((
-                "Cannot read 0x value with length %d. Expecting 42 (address "
+                f"Cannot read 0x value with length {len(orignum)}. Expecting 42 (address "
                 "incl 0x) or 66 (bytes32 incl 0x)"
```

Comment:
Combined with above and below
```suggestion
                "incl 0x) or 66 (bytes32 incl 0x)",
```